pkg/gui/controllers/branches_controller.go

```diff
@@ -764,6 +776,28 @@ func (self *BranchesController) createPullRequestMenu(selectedBranch *models.Bra
 	return self.c.Menu(types.CreateMenuOptions{Title: fmt.Sprint(self.c.Tr.CreatePullRequestOptions), Items: menuItems})
 }
 
+func (self *BranchesController) promptForTargetBranchNameAndCreatePullRequest(fromBranch *models.Branch, toRemote string) error {
+	_, ok := lo.Find(self.c.Model().Remotes, func(remote *models.Remote) bool {
```

Comment:
Better to use [lo.NoneBy](https://github.com/samber/lo#noneby) here

Reply:
Updated, thanks for the hint.